### DISCUSSION TITLE
Can you modify the style of a component completely to match your own requirements?

### DISCUSSION THREAD

**Discussion by hamidhosseini** [quasarframework/quasar#11291] [General - Components / Directives / etc]
Hi everyone,

at my work, we are about to start a new project and we are considering Quasar. The problem is that our company has it's own style guides so the UI element look completely different to what Quasar components has to offer. I wanted to know if it's possible to change the style of a component completely to match it to our company requirements.

**Comment by hawkeye64** [comment]
Yes, if you are confident in your CSS skills. Many of the components have a `flat` attribute so allows you full control. Many more controls will be like this in the future. In the meantime, for those that are not, just override the CSS.

Look at QBtnGroup here: https://quasar.dev/vue-components/button-group#usage
Then look how I styled it here: https://codepen.io/Hawkeye64/pen/WNNdKJV
